os/arch/arm/src/armv7-r/mpu.h

```diff
@@ -909,13 +920,14 @@ static inline void mpu_user_intsram_ro(uintptr_t base, size_t size)
  *
  ****************************************************************************/
 
-static inline void mpu_user_intsram_wb(uintptr_t base, size_t size)
+static inline void mpu_user_intsram_wb(uint32_t region, uintptr_t base, size_t size)
 {
-	unsigned int region = mpu_allocregion();
```

Comment:
I think you removed all of mpu_allocregion used. But not removed that function implementation. Could you check that?

Reply:
The allocregion function seems to be a utility api. 
I have not deleted it since we might use it in future for some purpose.